src/Emit.ts

```diff
@@ -17,7 +15,7 @@ export function emitTranspiledFiles(
     transpiledFiles: TranspiledFile[],
     emitHost: EmitHost = ts.sys
 ): OutputFile[] {
-    let { rootDir, outDir, outFile, luaLibImport } = options;
+    let { rootDir, outDir, luaLibImport } = options;
 
     const configFileName = options.configFilePath as string | undefined;
     // TODO: Use getCommonSourceDirectory
```

Comment:
Can't we use `getProjectRootDir` now? (for `rootDir = rootDir || baseDir`, can't leave comment on that line)